src/blocks/scratch3_sound.js

```diff
@@ -88,6 +93,23 @@ class Scratch3SoundBlocks {
         return soundState;
     }
 
+    /**
+     * When a Target is cloned, clone the sound state.
+     * @param {Target} newTarget - the newly created target.
+     * @param {Target} [sourceTarget] - the target used as a source for the new clone, if any.
+     * @listens Runtime#event:targetWasCreated
+     * @private
+     */
+    _onTargetCreated (newTarget, sourceTarget) {
+        if (sourceTarget) {
+            const soundState = this._getSoundState(sourceTarget);
```

Comment:
This `_getSoundState` call will create the state on `sourceTarget` if it doesn't exist. That's not a huge problem but I'd consider avoiding that just so that this code doesn't modify targets which don't use these blocks.

Reply:
Good catch!